Document modifyMedia for lib-content #455 (#462)
enonic/doc-xp: docs/api/lib-content.adoc

};
```

=== modifyMedia

Modifies a media content

[.lead]
*Parameters:*

An object with the following keys and their values:

[%header,cols="1%,1%,1%,1%,98%a"]
[frame="none"]
[grid="none"]
|===
| Name | Type | Attributes| Default| Description

> | key | string | | | Path or id to the content

Reviewer: ```suggestion
| key | string | | | Path or id of the media content
```

> | name | string | | | Name of content

Reviewer: ```suggestion
| name | string | | | Name of the media content
```

> | data | | | | Data (as stream) to use

Reviewer: ```suggestion
| data | | | | Media data (as a stream)
```

| mimeType | string | <optional> | | Mime-type of the data

> | focalX | number | <optional> | | Focal point for X axis (if it's an image)

Reviewer: ```suggestion
| focalX | number | <optional> | | Focal point for X axis (if content is an image)
```

> | focalY | number | <optional> | | Focal point for Y axis (if it's an image)

Reviewer: ```suggestion
| focalY | number | <optional> | | Focal point for Y axis (if content is an image)
```

> | caption | string | <optional> | | Caption to the content

Reviewer: ```suggestion
| caption | string | <optional> | | Caption
```

> | artist | string \| Array.<string>  | <optional> | | Artist to the content

Reviewer: ```suggestion
| artist | string \| Array.<string>  | <optional> | | Artist
```

> | copyright | string | <optional> | | Copyright to the content

Reviewer: ```suggestion
| copyright | string | <optional> | | Copyright
```

> | tags | string \| Array.<string> | <optional> | | Tags to the content

Reviewer: ```suggestion
| tags | string \| Array.<string> | <optional> | | Tags
```

> | workflowInfo | object | <optional> | | Workflow information to use. Default has state READY and empty check list.

Reviewer: ```suggestion
| workflowInfo | object | <optional> | | Workflow state (default: READY).
```
